app/views/shared/_svg_sprite.html.erb

```diff
@@ -0,0 +1,12 @@
+<%
+  # Include this once a page then reference the symbols where you want them eg:
+  #
+  # <svg>
+  #   <use xlink:href="#svg-arrow-right"></use>
+  # </svg>
+%>
+<svg style="display: none;">
+  <symbol id="svg-arrow-right" viewBox="0 0 22 34">
+    <polygon points="0,5 5,0 22,17 5.1,34 0,29 12,17" />
+  </symbol>
+</svg>
```

Comment:
Unfortunately, no version of IE supports referencing external svgs with `use`. There is a polyfill: https://github.com/jonathantneal/svg4everybody